sig-docs/localization/ZH-CN Localization Task.md

```diff
@@ -113,6 +113,13 @@ Please read the following rules before you claim a task:
 | [Set Email Server for KubeSphere Pipelines](https://kubesphere.io/docs/devops-user-guide/how-to-use/jenkins-email/) | [Kai Zhang](https://github.com/mvpzhangkai) | [620](https://github.com/kubesphere/website/pull/620) |
 | [Integrate SonarQube into Pipeline](https://kubesphere.io/docs/devops-user-guide/how-to-integrate/sonarqube/) | [Kai Zhang](https://github.com/mvpzhangkai) |         |
 | [Choose Jenkins Agent](https://kubesphere.io/docs/devops-user-guide/how-to-use/choose-jenkins-agent/) | [Kai Zhang](https://github.com/mvpzhangkai) |         |
+| [Create a Pipeline Using a Jenkinsfile](https://kubesphere.io/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile/) | [Kai Zhang](https://github.com/mvpzhangkai) |         |
+| [DevOps Project Management](https://kubesphere.io/docs/devops-user-guide/how-to-use/devops-project-management/) | [Kai Zhang](https://github.com/mvpzhangkai) |         |
+| [Role and Member Management](https://kubesphere.io/docs/devops-user-guide/how-to-use/role-and-member-management/) | [Kai Zhang](https://github.com/mvpzhangkai) |         |
+| [Create a Pipeline - using Graphical Editing Panel](https://kubesphere.io/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel/) | [Kai Zhang](https://github.com/mvpzhangkai) |  |
+| [Choose Jenkins Agent](https://kubesphere.io/docs/devops-user-guide/how-to-use/choose-jenkins-agent/) | [Kai Zhang](https://github.com/mvpzhangkai) |         |
```

Comment:
Delete this please. Choose Jenkins Agent already exists. Thx.

Reply:
OK ，These articles have been deleted。